### PR TITLE
Fix issue with Google Translate

### LINKED ISSUE #2260: When auto-translating docs, code is also translated
Page URL: https://dart.dev/tutorials/server/cmdline
Page source: https://github.com/dart-lang/site-www/tree/master/src/_tutorials/server/cmdline.md

Found a typo? You can fix it yourself by going to the page source and clicking the pencil icon. Or finish creating this issue.

Description of issue: при нажатии перевести на русский так же переводится и код языка, и что это за учебник который говорит скопируйте вот эту простыню кода, теперь у вас все работает, круто да? вам его писал тот же чувак что писал Тур Героев? полный атстой

### PR DESCRIPTION
**Pull request by johnpryan**
Google translate will process any `<pre>` tags unless they include a `<code>` tag too. This fixes any snippets generated by `create_code_with_tooltips.dart` and all code snippets using the `prettify` Liquid plugin.

fixes #2260

### CODE REVIEW

**Comment by johnpryan** [comment]
This doesn't fully fix #2260 yet... investigating.

**Comment by johnpryan** [comment]
This is a big change, so I've deployed them here:  https://jr-dartlang-1.firebaseapp.com - we should probably kick the tires a bit. I've been using the Google Translate chrome extension to test

**Comment by kwalrath** [comment]
Didn't see your comment, @johnpryan :). 

@MitaiGit can you confirm that this fixes #2260?

File: src/_tutorials/server/_dcat-example.html [hunk 1]
```diff
@@ -4,6 +4,7 @@
 from sources in the example folder.
 {% endcomment -%}
 <pre class="prettyprint lang-dart">
```
Comment (kwalrath) [inline src/_tutorials/server/_dcat-example.html]: Change from `<code>` to `tag=pre+code`?

**Comment by kwalrath** [comment]
When I compare the original (https://dart.dev/tutorials/server/cmdline#overview-of-the-dcat-app-code) to my staged version (https://kw-www-dartlang-1.firebaseapp.com/tutorials/server/cmdline#overview-of-the-dcat-app-code), I see extra space above the first line (`import 'dart:convert';`) in the staged version.

I don't see this anywhere else, so I think it's just a buglet in the included file (src/_tutorials/server/_dcat-example.html).

I'll continue looking...

**Comment by kwalrath** [comment]
The samples page is much better with these changes, hooray! (I compared https://jr-dartlang-1.firebaseapp.com/samples#control-flow-statements to https://dart.dev/samples#control-flow-statements, using the Google Translate extension to translate both pages to Spanish.)

**Comment by kwalrath** [comment]
Everything I've spot-checked LGTM with the exception of that one bit of extra space in the command-line tutorial (not a big deal).

I say we ship it.

File: src/_tutorials/server/_dcat-example.html (hunk 1 above)
Comment (johnpryan) [reply src/_tutorials/server/_dcat-example.html]: hmm, I don't think that's required here. (This file is generated by the script.)

**Comment by MitaiGit** [comment]
Снимок экрана от 2020-02-08 22-50-05

**Comment by johnpryan** [comment]
@MitaiGit this hasn't been released yet, hang tight.

**Comment by kwalrath** [comment]
We can fix the extra space in the command-line tutorial later.

**Comment by kwalrath** [comment]
Confirmed that the website has been updated. @MitaiGit please let us know if you find any other issues.

**Comment by MitaiGit** [comment]
> Подтвердили, что сайт обновлен. @ MitaiGit, пожалуйста, дайте нам знать, если вы обнаружите какие-либо другие проблемы.

https://dart.dev/tutorials/language/streams
Снимок экрана от 2020-02-10 22-55-45

**Comment by kwalrath** [comment]
Could you try in an incognito window (or empty the cache another way)? The code in that page is working for me (although it looks like we should code font **await for** in the text):

image

**Comment by MitaiGit** [comment]
there is a more interesting fragment, it is partially translated
Снимок экрана от 2020-02-10 23-05-23

**Comment by kwalrath** [comment]
I wonder why non-code-font words are sometimes translated and sometimes not. Also, I wonder if there's a way to mark a word as untranslated without making it code font (which is ugly and takes a lot of space, so we try to avoid it when it's clear that we're talking about API/keywords).

/cc @jcollins-g @kevmoo @jakemac53

**Pull request merged**